dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -1,15 +1,28 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 import "..\AppNotifications\AppNotifications.idl";
 
 namespace Microsoft.Windows.AppNotifications.Builder
 {
-    [contractversion(1)]
+    [contractversion(2)]
     apicontract AppNotificationBuilderContract {}
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationHeader
+    {
+        AppNotificationHeader();
+        AppNotificationHeader(String id, String title);
+
+        String Id;
+        String Title;
+        Windows.Foundation.Collections.IMap<String, String> Arguments;
+
+        AppNotificationHeader AddArgument(String key, String value);
+    };
+
     [contract(AppNotificationBuilderContract, 1)]
-    runtimeclass AppNotificationTextProperties
+    unsealed runtimeclass AppNotificationTextProperties
```

Comment:
Why `unsealed`?

Why `unsealed` here but not say `AppNotificationHeader` (line 12)?